.gitignore

```diff
@@ -1,2 +1,5 @@
 /target
 Cargo.lock
+.envrc
```

Comment:
I believe that OS-specific things belong to global .gitignore, would like to keep this .gitignore for rust-only things